api/client/client.go

```diff
@@ -3441,6 +3441,16 @@ func (c *Client) UpdateClusterMaintenanceConfig(ctx context.Context, cmc types.C
 	return trail.FromGRPC(err)
 }
 
+// GetFeatures returns the feature set of the cluster.
+func (c *Client) GetFeatures(ctx context.Context) (*proto.Features, error) {
+	req, err := c.grpc.GetFeatures(ctx, &emptypb.Empty{})
+	if err != nil {
+		return nil, trail.FromGRPC(err)
+	}
+
+	return req.GetFeatures(), nil
+}
```

Comment:
> I've created a new endpoint instead of using Ping, which already gets the features, to avoid overloading this endpoint too much. Thoughts?

Why do you mean by "overloading" ? 
As far I can see the Ping endpoint is lite  and calling Ping vs Features endpoint don't make any difference in terms of pref.  

If we can reuse Ping API endpoint I think that we should do it.